octopus-server/src/main/resources/buildServerResources/v2/subpages/editCreateDeploymentParameters.jsp

```diff
@@ -0,0 +1,52 @@
+<%@ include file="/include-internal.jsp" %>
+<%@ taglib prefix="props" tagdir="/WEB-INF/tags/props" %>
+<%@ taglib prefix="forms" tagdir="/WEB-INF/tags/forms" %>
+<%@ taglib prefix="l" tagdir="/WEB-INF/tags/layout" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<%--
+  ~ Copyright 2000-2021 Octopus Deploy Pty. Ltd.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  --%>
+
+<jsp:useBean id="params" class="octopus.teamcity.server.generic.BuildStepCollection"/>
+<jsp:useBean id="keys" class="octopus.teamcity.common.createdeployment.CreateDeploymentPropertyNames"/>
+<jsp:useBean id="propertiesBean" scope="request" type="jetbrains.buildServer.controllers.BasePropertiesBean"/>
+
+<l:settingsGroup title="Create Deployment">
+    <tr>
+        <th>Project name/ID:<l:star/></th>
```

Comment:
personal statement: I hate name/ID :/ But will try not to die on this hill

Reply:
I agree it isn't great, will see what conversations about only supporting name yield in terms of changes

Reply:
Decision is to move this to "name" only. However adding support for create-deployment is now on hold